preprocessing-service/preprocess.py

```diff
@@ -175,18 +173,24 @@ async def mask_logs(queue):
             payload_data_df["kubernetes_component"] = payload_data_df[
                 "kubernetes.labels.component"
             ]
+        masked_logs = []
```

Comment:
why moving those 4 lines from 115 to here?

Reply:
That way, I can apply the masking after it is determined which log is a control plane log.